From: Chris Hofstaedtler <zeha@debian.org>
Date: Tue, 18 Jul 2023 23:34:51 +0200
Subject: fincore: (tests) fix double log output

The fincore tests call ts_log_both inside an output redirection of both stdout
and stderr, leading to the ts_log_both output ending up twice in both stdout
and stderr.
Call ts_log_both before setting up the output redirection.

Signed-off-by: Chris Hofstaedtler <zeha@debian.org>
---
 tests/ts/fincore/count | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tests/ts/fincore/count b/tests/ts/fincore/count
index ba5f6cd..c89ad9f 100755
--- a/tests/ts/fincore/count
+++ b/tests/ts/fincore/count
@@ -119,11 +119,11 @@ check_dd_fs_feat
 INPUT=
 input=
 
+ts_log_both "[ NO EXCITING FILE ]"
 {
     input=no_such_file
     INPUT="${INPUT} ${input}"
 
-    ts_log_both "[ NO EXCITING FILE ]"
     $TS_CMD_FINCORE --output $OUT_COLUMNS --bytes --noheadings $input
     footer "$?"
 } >> $TS_OUTPUT 2>> $TS_ERRLOG
@@ -220,8 +220,8 @@ input=
 		   "oflag=append seek=$hole_count"
 } >> $TS_OUTPUT 2>> $TS_ERRLOG
 
+ts_log_both "[ MULTIPLE FILES ]"
 {
-    ts_log_both "[ MULTIPLE FILES ]"
     $TS_CMD_FINCORE --output $OUT_COLUMNS --bytes $INPUT
     footer "$?"
 } >> $TS_OUTPUT 2>> $TS_ERRLOG
